spring-beans/README.md

```diff
@@ -34,3 +34,36 @@ Here are some example beans using the factories in this module:
     <property name="tracing" ref="tracing"/>
   </bean>
 ```
+The factory beans can also be used in java configuration:
```

Comment:
why wouldn't you just use normal beans?

Reply:
the normal beans will not be closed() automatically by Spring unless i misunderstood, to be honest i did not know about the factory beans until you mentioned them here https://github.com/openzipkin/brave/pull/507.